modules/navier_stokes/src/fvkernels/INSFVMomentumDiffusion.C

```diff
@@ -38,9 +46,14 @@ INSFVMomentumDiffusion::INSFVMomentumDiffusion(const InputParameters & params)
 ADReal
 INSFVMomentumDiffusion::computeStrongResidual()
 {
-  const auto face = Moose::FV::makeCDFace(*_face_info, faceArgSubdomains());
   const auto dudn = gradUDotNormal();
-  const auto face_mu = _mu(face);
+  ADReal face_mu;
+  Moose::FV::interpolate(_mu_interp_method,
+                         face_mu,
+                         _mu(elemFromFace()),
+                         _mu(neighborFromFace()),
+                         *_face_info,
+                         true);
```

Comment:
My heart prefers that we perhaps make another `makeFace` overload and then keep the call to the functor with the `FaceArg`.

But my head says this is a shorter function call stack and consequently maybe faster.

Reply:
the cool thing with writing it like this is that it looks the same for k and eps * k for example